api4/cloud.go

```diff
@@ -41,6 +41,9 @@ func (api *API) InitCloud() {
 	api.BaseRoutes.Cloud.Handle("/subscription/invoices/{invoice_id:in_[A-Za-z0-9]+}/pdf", api.APISessionRequired(getSubscriptionInvoicePDF)).Methods("GET")
 	api.BaseRoutes.Cloud.Handle("/subscription", api.APISessionRequired(changeSubscription)).Methods("PUT")
 
+	// GET /api/v4/cloud/integrations
+	api.BaseRoutes.Cloud.Handle("/installed_integrations", api.APISessionRequired(getInstalledIntegrations)).Methods("GET")
```

Comment:
@neallred `/api/v4/cloud` may not be the right spot for this (also I see I have a typo in the comment). At the moment, is no general purpose subpath for querying integrations. They are currently segmented between `api/v4/plugins` and `/plugins/com.mattermost.apps`. I'm not sure if this endpoint warrants creating a `api4/integrations.go` file.

Reply:
> , but at the same time this is a request specific to cloud instances of Mattermost, so it makes sense here, too.

It seems `getInstalledIntegrations` doesn't seem to have anything specific to cloud or CWS requests. 2/5 that it would make sense more under a new `/limits` subpath or even under a more integrations related endpoint. This information will still be useful in other scenarios outside of cloud like for example an Insights type of dashboard in System Console